fvm/evm/handler/blockstore.go

```diff
@@ -47,7 +47,19 @@ func (bs *BlockStore) BlockProposal() (*types.BlockProposal, error) {
 	if len(data) != 0 {
 		return types.NewBlockProposalFromBytes(data)
 	}
+	bp, err := bs.constructBlockProposal()
```

Comment:
this should only execute for the genesis block right?

Reply:
genesis or first time we deploy this change to Testnet